benchmark-service/MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include benchmark/templates *
```

Comment:
Can you add a new line at the end of the file? Just want to get rid of the red minus/circle symbol here.